aws/sqs_event_manager/sqs_event_manager/app.py

```diff
@@ -106,13 +106,10 @@ def process_message(record: dict, batch_item_failures: dict):
                 return error_response(400, entitlements.error)
 
             request = {  # noqa TODO: Cleanup noqa
-                'isBase64Encoded': False,
-                'statusCode': 200,
-                'body': {
-                    'CustomerIdentifier': message.customer_id,
-                    'ProductCode': message.product_code,
-                    'Entitlements': entitlements.get_entitlements()
-                }
+                'CustomerIdentifier': message.customer_id,
+                'MarketplaceIdentifier': 'AWS',
+                'ProductCode': message.product_code,
+                'Entitlements': entitlements.get_entitlements()
```

Comment:
In agreement with Alex and according to the swagger the key names should be:

```
marketplaceIdentifier
customerIdentifier
productCode
entitlements
```

So they all start with a lowercase letter. Can you update that so we are consistent.

After this all is good.

Thanks